docs/man/futhark.rst

```diff
@@ -66,6 +66,12 @@ A Futhark compiler development command, intentionally undocumented and
 intended for use in developing the Futhark compiler, not for
 programmers writing in Futhark.
 
+futhark eval options... <exprs...>
+----------------------------------
+
+Evaluates expressions given as command-line arguments. Optionally 
+allows a file import.
```

Comment:
Document the syntax for the file import.  There are few enough options that you can just spell them out (instead of `options...` put `[-f FILE] [-w]`, or whatever it is).

Reply:
added the individual options